test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -262,6 +263,28 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #region Join tests
 
+        [Test]
+        [TestModel(@".\elementJoin.rvt")]
+        public void CanSuccessfullyGetJoinedElementsFromElement()
+        {
+            // Arrange - get element from model
+            var element = ElementSelector.ByElementId(184176, true);
+
+            // Act
+            List<Element> joinedElements = element.GetJoinedElements();
+
+            // Assert
+            Assert.AreEqual(3, joinedElements.Count);
+
+            List<int> joinedElementIds = new List<int>();
+            for (int i = 0; i < joinedElements.Count; i++)
+            {
+                joinedElementIds.Add(joinedElements[i].Id);
+            }
```

Comment:
can be replaced with `var joinedElementIds = joinedElements.Select(x=>x.Id).ToList();` - much more readable than a `for` loop and performance hit isn't a factor as it's a test. do this for the other tests too

Reply:
this should also be moved to `Act`